### DISCUSSION TITLE
New maintainers

### DISCUSSION THREAD

**Discussion by Echolon** [grote/Transportr#906] [General]
Looking at the recent contributions and PRs would it make sense to give more rights to @Altonss? (if wanted)

@grote @ialokim

**Comment by Altonss** [comment]
Thanks @Echolon for your message :) I already have enough rights to merge PRs and deal with issues!

I'm just waiting to get some merged because I don't know if I need at least the approval of another maintainer... From past contributions (from @ialokim for example), I see that one maintainer is enough (see #884). But to be sure I'm waiting. It would be nice to have some feedback of @ialokim and/or @grote on this?

**Reply by ialokim** [reply]
Indeed, I think it's a good rule to wait for an approval or comment of at least one other maintainer.

I'm sorry for being quite absent again lately, but I'll try to dedicate some time to Transportr over the weekend to finally get the new version out.

**Reply by Altonss** [reply]
No worries :) Thanks for the comment, I think it's a good rule to follow!

**Comment by Echolon** [comment]
Alright, sorry for confusion. I will close here.
